### DISCUSSION TITLE
Docker / Containerization

### DISCUSSION THREAD

**Discussion by jarmentor** [gbowne1/TwitchBot#76] [Ideas]
Greetings,

It seems to me there's quite a few moving pieces at least when working, a mongodb instance, a front-end dev server, a backend server that hits the mongodb instance (am I understanding this correctly). If so, it also occurs to me that containerization might be useful to quickly run things?

I'm only what one might say is marginally familiar with this, I've set it up on a handful of things. What say you all?

Also noting that this may be touched on in #75

**Comment by gbowne1** [comment]
Great idea @jarmentor
